config/environments/development.rb

```diff
@@ -2,7 +2,11 @@
   # Verifies that versions and hashed value of the package contents in the project's package.json
   config.webpacker.check_yarn_integrity = true
 
-  config.action_controller.asset_host = Figaro.env.domain_name
+  config.action_controller.asset_host = proc do |source, request|
+    # we want precompiled assets to have domain-agnostic URLs
+    # and request is nil during asset precompilation
+    Figaro.env.domain_name if request
+  end
```

Comment:
I think we don't need this, I didn't actually get this working locally, I think webpacker stuff may not use this? Or it was a caching issue? Anyways we really only care about this for prod anyways